src/uast.cc

```diff
@@ -266,7 +266,7 @@ Nodes *UastFilter(const Uast *ctx, void *node, const char *query) {
     nodes = new Nodes();
   } catch(const std::bad_alloc&) {
     Error(nullptr, "Unable to get memory for nodes\n");
-    return nullptr;
+    throw std::runtime_error("");
```

Comment:
Keep this, for the reasons explained below.

Reply:
👍 Changes made!